filament/src/details/Engine.cpp

```diff
@@ -646,6 +646,15 @@ int FEngine::loop() {
     uint32_t id = std::thread::hardware_concurrency() - 1;
 
     while (true) {
+        #if FILAMENT_ENABLE_FGDBG
+              if (debug.fgdbg) {
+                static int fgdbgMessage = 0;
+                fgdbgMessage++;
+                if (fgdbgMessage % 100 == 0) {
+                  debug.fgdbg->sendMessage(fgdbgMessage/100);
+                }
+              }
+        #endif
```

Comment:
Just for testing purposes, we can do without this. Let me know if should take this out before merge